packages/control-property-editor/src/components/ModeSwitcher.scss

```diff
@@ -0,0 +1,16 @@
+ .mode-switcher {
+    display: flex;
+    align-items: center;
+
+    .ms-Button { 
+        border-radius: 3px;
+        margin-right: 1px;
+        &-label {
+            font-weight: bold;
+        }
+    }
+ 
+    .ms-Label {
+        margin-right: 10px;
+    }
+} 
```

Comment:
Are we using BEM methodology here anywhere in the CPE styles code? Should we enforce it here or are the styles are not that many?

Reply:
So far we do not have that many styles in CPE, so we do not have any strict rules on how to structure the CSS selectors. If the number of styles and selectors grows, we may need to revisit this topic.